lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -122,8 +122,8 @@ SmallVector<unsigned> getContigPerThread(Attribute layout) {
   }
 }
 
-SmallVector<unsigned> getUniqueContigPerThread(Attribute layout,
-                                               ArrayRef<int64_t> shape) {
+SmallVector<unsigned> oldGetUniqueContigPerThread(Attribute layout,
```

Comment:
Is it still used? If not, why not taking it out?

Reply:
nope, I'll kill it. I was using it to debug